dephell/constants.py

```diff
@@ -79,27 +79,38 @@ class ReturnCodes(Enum):
 VERSION_MINOR = ('minor', 'feature')
 VERSION_PATCH = ('patch', 'fix', 'micro')
 VERSION_PRE = ('pre', 'rc', 'alpha')
+VERSION_PRE_MAJOR = ('premajor', 'prebreaking')
+VERSION_PRE_MINOR = ('preminor', 'prefeature')
+VERSION_PRE_PATCH = ('prepatch', 'prefix', 'premicro')
+PRE_VERSIONS = VERSION_PRE + VERSION_PRE_MAJOR + VERSION_PRE_MINOR + VERSION_PRE_PATCH
+MAJOR_VERSIONS = VERSION_MAJOR + VERSION_PRE_MAJOR
+MINOR_VERSIONS = VERSION_MINOR + VERSION_PRE_MINOR
+PATCH_VERSIONS = VERSION_PATCH + VERSION_PRE_PATCH
+VERSION_RELEASE = ('release', )
 # semver has no post-releases: https://github.com/semver/semver/issues/200
 VERSION_POST = ('post', )
 VERSION_DEV = ('dev', )
 VERSION_LOCAL = ('local', )
 VERSION_SCHEMES = MappingProxyType(dict(
     # https://www.python.org/dev/peps/pep-0440/#version-scheme
-    pep=VERSION_MAJOR + VERSION_MINOR + VERSION_PATCH + VERSION_PRE + VERSION_POST + VERSION_DEV + VERSION_LOCAL,
+    pep=(
```

Comment:
@orsinium this is going to fail flake8 (at least it did locally).  Can't have a binary operator immediately before or after a line break, but without line breaks the line is too long.  I can fake out flake8 by adding the + after the closing parens on line 99, but that is cheap.

Struggling with things like this is why I want an auto-formatter to save me time 😄.  What should I do?  Get more clever with how I bundle the version combos (like I did for `MAJOR_VERSIONS`)?